.github/workflows/slash-command-merge.yml

```diff
@@ -52,6 +52,7 @@ jobs:
             }
 
             let [ body, body_tail ] = pr_info.body.split(/^---\s*$/m, 2);
+            body = body.replace(/<\!--.*?-->/sg, "");
```

Comment:
```suggestion
            body = body.replace(/<\!--.*?-->/s, "");
```

my thinking is that if we drop the `g`, it will only replace the first occurence, avoiding the potential pitfall that @alaviss mentioned. Instead if we later add more `<!-- blah -->` sections those won't be covered, but having them in the middle is likely not the way to go anyway.

Reply:
@Clyybber bump